frontend/jest.setup.ts

```
@@ -5,15 +5,15 @@ import React from "react";
 const originalLocation = window.location;
 
 beforeAll(() => {
-  delete window.location;
+  delete (window as any).location;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)